### PR TITLE
Display any existing translation text in the editor

### LINKED ISSUE #29: Editor text box shows the wrong text
State isn't being correctly updated somewhere, so the text box shows the last field edited when I go to edit a new field.

eg. here I'm editing the project description, after editing the title, but the text input still shows the project title.
Screenshot of the editor dialog showing the wrong text being displayed in the text input

### PR DESCRIPTION
**Pull request by simoneduca**
Fix #29

### CODE REVIEW

**Comment by eatyourgreens** [comment]
Doesn't this just show an empty box in the input field when it should show the existing translation?

File: src/containers/ProjectContentsContainer.jsx [hunk 1]
```diff
@@ -125,7 +131,6 @@ class ProjectContentsContainer extends Component {
           onChange={this.handleChange}
           placeholder="Translate some text"
           type="text"
-          value={this.state.translationText}
         />
```
Comment (eatyourgreens) [inline src/containers/ProjectContentsContainer.jsx]: This isn't setting a value for the input field, even if there is text in the translation.

File: src/containers/ProjectContentsContainer.jsx [hunk 2]
```diff
@@ -103,6 +106,9 @@ class ProjectContentsContainer extends Component {
       actions.updateTranslation(translation, field, translationText);
     }
     this.closeModal();
+    this.setState({
+      translationText: '',
```
Comment (eatyourgreens) [inline src/containers/ProjectContentsContainer.jsx]: This means that the new translation will never display in the editor, right? That's wrong.

**Comment by simoneduca** [comment]
I didn't realise the original translation is supposed to show.

**Comment by eatyourgreens** [comment]
If you've entered text already, I think you'd expect it to show if you clicked to edit it again eg. if you'd made a mistake.

File: src/containers/ProjectContentsContainer.jsx [hunk 3]
```diff
@@ -105,17 +109,17 @@ class ProjectContentsContainer extends Component {
     this.closeModal();
   }
 
-  renderInput() {
-    const { supportsMarkdown } = this.state;
+  renderInput(translation) {
```
Comment (eatyourgreens) [inline src/containers/ProjectContentsContainer.jsx]: Does this `translation` argument get used in this function?

**Pull request merged**